.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.2.0",
-  "packages/api-explorer": "0.9.29",
+  ".": "2.3.0",
+  "packages/api-explorer": "0.9.30",
   "packages/code-editor": "0.1.19",
-  "packages/extension-api-explorer": "22.2.1",
-  "packages/extension-sdk": "22.2.0",
-  "packages/extension-sdk-react": "22.2.1",
-  "packages/extension-utils": "0.1.6",
-  "packages/hackathon": "22.2.1",
-  "packages/run-it": "0.9.29",
-  "packages/sdk": "22.2.0",
+  "packages/extension-api-explorer": "22.4.0",
```

Comment:
good, now we see all the correct "22.4.0" versions

Reply:
Yup! I've cut an issue the possible bug https://github.com/googleapis/release-please/issues/1348